db/migrate/20151127120844_add_prisoner_model.rb

```diff
@@ -0,0 +1,33 @@
+class AddPrisonerModel < ActiveRecord::Migration
+  def up
+    create_table :prisoners, id: :uuid, default: 'uuid_generate_v4()' do |t|
```

Comment:
As the PG uuid module is activated, so you don't need the default: 

`create_table :prisoners, id: :uuid do |t|`